src/main.cpp

```diff
@@ -202,8 +202,20 @@ main(int argc, char *argv[])
     QCommandLineParser parser;
     parser.addHelpOption();
     parser.addVersionOption();
-    QCommandLineOption debugOption(QStringLiteral("debug"), QStringLiteral("Enable debug output"));
+    QCommandLineOption logLevel(
+      QStringLiteral("loglevel"),
+      QStringLiteral("Set the log level, or a <component>=<level> list like \"warn,ui=off\". "
```

Comment:
```suggestion
      QStringLiteral("Specify the log levels. You can set the global log level or the log level for individual components or a combination of both. For example to set the default log level to warn but disable logging for the ui component, pass '--log-level=warn,ui=off'."
```

Reply:
Thanks. We need a trailing space at the end of that sentence, though. Does github let you edit your diff? If not, I can make the change locally and include it in my next update.